services/web/client/source/class/qxapp/component/workbench/NodeUI.js

```diff
@@ -130,11 +135,17 @@ qx.Class.define("qxapp.component.workbench.NodeUI", {
         width: "50%"
       });
 
-      this.__progressBar = new qx.ui.indicator.ProgressBar().set({
-        height: 10,
-        margin: 4
-      });
-      this.add(this.__progressBar);
+      this.add(this.__createChipContainer());
+
+      if (this.getNode().getMetaData().category === "Solver") {
```

Comment:
this.getNode().isComputational() ?

Reply:
This is only for solvers... The file picker is also set as computational (maybe we change that in the future), but only solvers will actually show a progress bar

Reply:
As far as I know, we don't have Computational services other than Solvers. Am I missing something?

Reply:
Alright